docs/players-and-controllers/index.md

```diff
@@ -3,33 +3,22 @@ layout: default
 title: Overview
 ---
 
-## Hardware based
-
-Between 2001 and 2010 Logitech (and its predecessor Slim Devices) released a series of network music players. The lineup consisted of the Squeezebox Classic, an all-in-one Squeezebox Boom, the dual unit Squeezebox Duet, the audiophile grade Transporter, the small and optionally battery powered Squeezebox Radio, and the newest of the bunch, the Squeezebox Touch.
-
-Additionally, the Squeezebox Controller provides control of all your players via an attractive full color display. The Controller and Receiver were sold together as the Squeezebox Duet package.
-
-- [Hardware comparison](hardware-comparison.md)
-- [Squeezebox Touch](squeezebox-touch.md)
-- [Squeezebox Radio](squeezebox-radio.md)
-- [Squeezebox Boom](squeezebox-boom.md)
-- Squeezebox Duet
-    - [Squeezebox Controller](squeezebox-controller.md)
-    - [Squeezebox Receiver](squeezebox-receiver.md)
-- [Transporter](transporter.md)
-- [Squeezebox 3rd Generation](squeezebox-classic.md)
-- [Squeezebox2](squeezebox2.md)
-- [Squeezebox, SB1](squeezebox1.md)
-- [SLIMP3](SLIMP3.md)
-
-The Radio, Touch and Controller software (operating system and application) is available as source code and have been modified by "the community".
-Software from Logitech: [Operating system](https://github.com/LMS-Community/squeezeos) - [Application](https://github.com/LMS-Community/squeezeplay)
-Community versions: [Operating system](https://github.com/ralph-irving/squeezeos)  - [Application](https://github.com/ralph-irving/squeezeos-squeezeplay)
-
-After 2010 the LMS community developed DIY hardware offerings.
-
-- [Squeezelite-ESP32](https://github.com/sle118/squeezelite-esp32)
-- [SqueezeAMP](https://github.com/philippe44/SqueezeAMP) - which is an implementation of Sqeezelite-ESP32
+## Hardware
+Although Slim Devices and Logitech players are no longer produced, there is a huge selection of hardware players that are available and compatible with LMS.  In fact, many users have moved on to do it yourself (DIY) hardware.  The DIY choices range from simple and free (install software on your computer or phone) to complex and expensive (reclockers with femtosecond accuracy, ultracapacitors, etc.)
+
+### Legacy Hardware
+Between 2001 and 2010 Logitech (and its predecessor Slim Devices) released [a series of network music players](hardware-comparison.md).  This hardware is still widely used (and treasured!) and used units can be found to purchase online. 
+ 
+### DIY hardware offerings
+DIY choices can range from relatively simple to complex, depending on your needs, budget, and skill level.  The examples below is not a complete list and is not an endorsement of one vs another, but an example of the range of products that are available.
+#### ESP32 based
+- [SqueezeAMP](https://github.com/philippe44/SqueezeAMP) running [Squeezelite-ESP32](https://github.com/sle118/squeezelite-esp32)
+#### Raspberry Pi based
```

Comment:
I think it would be good to add a paragraph or two describing what these options are, and how they work. Eg. the list of sound card vendors should not make users believe that those would sell ready made packages. It should clear that these are add-ons to a Pi, and that the Pi would have to be configured by the user (link to Getting Started).

Reply:
I can do that, but then it got me thinking about maybe there's a better way to organize the documentation.  I need to work through this a little bit, but here's an outline of what I'm proposing - would love feedback and suggestions.

1. Overview - what is not just LMS but the entire ecosystem.  What are the benefits / why is it important and different from other music player ecosystems?  (Just submitted [a pull request](https://github.com/LMS-Community/lms-community.github.io/pull/22) with some updates)
2. Getting Started - I think we need something before server download and installation.  Maybe an example architecture - here's a more detailed example of an easy way to get started, how to expand on that, and what an advanced system looks like?  I think jumping right into the server install is a bit much if we haven't given people an overview.
3. Then we go to server installation
4. Then we get to Players & Controllers and all the updates and information you're suggesting - I agree with you that we need more information here that can guide users on the options and how they work.
5. Then we get to Plugins and Customization
6. Maybe then we get to Advanced guides - OpenVPN on Touch/Radio, security concerns, etc.

Let me know what you think.

Reply:
I'd totally agree that this documentation has grown quickly (thanks everybody - in particular @terual!) but deservers a bit of reorg. In particular the Getting Started section. Would you want to do that as part of this PR, or a new PR?

Or better: let's outline in a thread on forums.slimdevices.com where we want to go, and coordinate among us so we don't risk to do things twice.